spec/models/user_spec.rb

```diff
@@ -317,6 +317,25 @@
     end
   end
 
+  describe '#visible_teams' do
```

Comment:
As said in my previous comment: I'd prefer to make the `visible` class method a scope in the `team` model. So you could probably move the specs over there if it's not tested accordingly yet.